spec/lib/task-data/schemas/dell-racadm-catalog-spec.js

```diff
@@ -10,10 +10,18 @@ describe(require('path').basename(__filename), function() {
         action: 'getConfigCatalog'
     };
 
+    var partialCanonical1 = {
+        action: 'getSoftwareList'
+    }
+
     var positiveSetParam = {
         action: ["getConfigCatalog"]
```

Comment:
You don't have to create another canonical options, just put the `getSoftwareList` in this `action` array will be OK.